internal/command/testdata/test/variables_reference/setup/main.tf

```diff
@@ -0,0 +1,17 @@
+variable "content" {
+  type = string
+}
+/*
```

Comment:
Why is this test commented out, could you write a comment about it please?

Reply:
Was an artifact of the test I used as reference.  Removed.